uuu/buildincmd.cpp

```diff
@@ -66,48 +64,14 @@ BuiltInScript::BuiltInScript(const BuiltInScriptRawData * const p) :
 	m_desc{p->m_desc ? p->m_desc : ""},
 	m_name{p->m_name ? p->m_name : ""}
 {
-	for (size_t i = 1; i < m_text.size(); i++)
-	{
-		size_t off;
-		if (m_text[i] == '_'
-			&& (m_text[i - 1] == '@' || m_text[i - 1] == ' '))
+	std::istringstream iss{m_text};
+	for (std::string line; std::getline(iss, line);) {
+		std::smatch arg_def_line_match;
+		if (std::regex_match(line, arg_def_line_match, arg_def_line))
 		{
```

Comment:
I am not sure regex_match exactly match original logic.

Reply:
I reduced the impact of my previous change and dropped the detection of argument descriptions and fallbacks back to the previous code. The regular expression should now match the previous behaviour exactly, but still be a bit more understandable.